core/src/main/java/org/elasticsearch/common/io/stream/StreamOutput.java

```diff
@@ -454,15 +454,28 @@ public void writeThrowable(Throwable throwable) throws IOException {
     /**
      * Writes a {@link NamedWriteable} to the current stream, by first writing its name and then the object itself
      */
-    public void writeNamedWriteable(NamedWriteable namedWriteable) throws IOException {
+    public void writeNamedWriteable(@SuppressWarnings("rawtypes") NamedWriteable namedWriteable) throws IOException {
         writeString(namedWriteable.getName());
         namedWriteable.writeTo(this);
     }
 
     /**
+     * Writes an optiona {@link NamedWriteable} to the current stream.
+     * */
+    public void writeOptionalNamedWriteable(@SuppressWarnings("rawtypes") NamedWriteable namedWriteable) throws IOException {
```

Comment:
alright, I do get unchecked warnings when it comes to generics, but not these ones, it is just the level of the warnings in the IDE. I agree we don't usually suppress these, I think we can remove.